packages/browser/test/e2e/test/utils/helpers.js

```diff
@@ -0,0 +1,61 @@
+const Sentry = require('@sentry/browser');
+
+function waitForXHR(xhr, cb) {
+  if (xhr.readyState === 4) {
```

Comment:
What does the `4` mean? If it's a standard way, can we add a comment to clarify it? If not, can we use a structure to make it easier to read and understand?

Reply:
Yes, those [XHR states](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState) might better live in an `enum`. 👍 Updating